tests/k8s/tests/01-guestbook-test.sh

```diff
@@ -15,6 +15,20 @@ dir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 
 source "${dir}/../cluster/env.bash"
 
+NAMESPACE="kube-system"
+TEST_NAME="01-guestbook-test"
+LOGS_DIR="${dir}/cilium-files/${TEST_NAME}/logs"
+
+function finish_test {
```

Comment:
Can't you make this a function in `tests/k8s/helpers.bash` that grabs the logs of all cilium instances running in k8s? we would still keep the `trap finish_test exit`.

`gather_k8s_logs` could do that, instead of specifying the cilium pod ID, the `gather_k8s_logs` would simply gather all logs from all cilium instances running in the cluster.